lib/gui/pages/finish/styles/_finish.scss

```diff
@@ -26,7 +26,9 @@
 }
 
 .page-finish .button-primary {
-  min-width: $btn-min-width;
```

Comment:
I think if we made _all_ the buttons have this same minimum width, then the three buttons of the main UI wouldn't fit into the window?